### ISSUE TITLE
cpr 1.6 does not compile with gcc 4.8

### ISSUE THREAD

**Issue by Renari** [whoshuu/cpr#541]
```
error: function ‘cpr::Authentication& cpr::Authentication::operator=(cpr::Authentication&&)’ defaulted on its first declaration with an exception-specification that differs from the implicit declaration ‘cpr::Authentication& cpr::Authentication::operator=(cpr::Authentication&&)’
     Authentication& operator=(Authentication&& old) noexcept = default;
```
Is output by the compiler, looks like the compiler is deleting these functions because they don't match.

**Comment by KingKili** [comment]
CPR is based on C++ 14, unfortunately gcc4.8 does not support C++14 entirely [https://en.cppreference.com/w/cpp/compiler_support/14](https://en.cppreference.com/w/cpp/compiler_support/14) .

**Comment by wzhe** [comment]
> CPR is based on C++ 14, unfortunately gcc4.8 does not support C++14 entirely https://en.cppreference.com/w/cpp/compiler_support/14 .

C++11 is written in the README.
What is the c++ version switching plan used by cpr?
Will the c++11 version be maintained? Even if it’s just a bug fixed, what’s the plan like, 1.5.3, 1.5.4, etc.
Thanks for your work, cpr is awesome!
